src/bun.js/api/server.zig

```diff
@@ -1465,8 +1465,6 @@ pub const ServerConfig = struct {
                                 args.ssl_config = ssl_config;
                             } else {
                                 if (ssl_config.server_name == null or std.mem.span(ssl_config.server_name).len == 0) {
-                                    var config = ssl_config;
-                                    defer config.deinit();
```

Comment:
This deinit looks safe to keep. It won't be assigned to `args.ssl_config` and it won't be appended to `args.sni`